packages/elements/src/combo-box/index.ts

```diff
@@ -1094,9 +1091,11 @@ export class ComboBox<T extends DataItem = ItemData> extends FormFieldElement {
       case 'Enter':
         this.enter(event);
         break;
+      case 'Up':
```

Comment:
Why are you adding legacy key value back in???
https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_key_values#navigation_keys